content/en/opentelemetry/agent/agent_with_custom_components.md

```diff
@@ -201,6 +207,8 @@ You can now use this new image to install the Agent. This enables Datadog monito
 
 For detailed instructions on installing and configuring the Agent with added OpenTelemetry components, see the [Install the Datadog Agent with Embedded OpenTelemetry Collector][9] guide.
 
+<div class="alert alert-info">The default configuration settings in Datadog's embedded collector may differ from the standard OpenTelemetry Collector configuration defaults. This can affect behavior of components like the <code>filelogreceiver</code>. Review the configuration closely when migrating from a standalone collector.</div>
+
```

Comment:
I believe this section should be on a different page: e.g. either a troubleshooting guide for "migrating to the DD agent with embedded OTel Collector" or a separate troubleshooting FAQ page, where we consolidate everything there. Will follow up via Slack thread.